src/styles/main.css

```diff
@@ -0,0 +1,79 @@
+
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  border: 1vmin #000;
+  border-style: dotted;
+  border-radius: 50%;
+  height: 80vmin;
+  width: 80vmin;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch__minutes {
+  position: absolute;
+  bottom: 50%;
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+  transform-origin: bottom center;
+  animation: rotation 3600s steps(60) infinite;
+}
+
+@keyframes rotation {
+  0% {
+    transform: rotate(0);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch__seconds {
+  position: absolute;
+  bottom: 50%;
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+  transform-origin: bottom center;
+  animation: rotation 60s infinite linear;
+}
+
+@keyframes rotation {
+  0% {
+    transform: rotate(0);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
The `@keyframes rotation` is defined twice in the file. Consider removing the duplicate definition to avoid redundancy and potential conflicts.